components/next_steps_view/steps.ts

```diff
@@ -67,7 +67,7 @@ export const Steps: StepType[] = [
         id: RecommendedNextSteps.NOTIFICATION_SETUP,
         title: localizeMessage(
             'next_steps_view.notificationSetup.setNotifications',
-            'Set up desktop notifications',
+            'Set up notifications',
```

Comment:
Is removing `desktop` here intended? Also, are we missing `next_steps_view.notificationSetup.setNotifications` from the localized strings?

Reply:
yep! `setNotifications` already exists,  and it reads `'Set up notifications'`, so making this match